hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/CandidateGenerator.java

```diff
@@ -28,6 +28,8 @@
 @InterfaceAudience.Private
 abstract class CandidateGenerator {
 
+  static double MAX_WEIGHT = 100_000;
```

Comment:
I thought it would make sense to have a universal ceiling for candidate generator weights, rather than them just being arbitrary doubles on any scale

Reply:
Admittedly, we could rip this out now that I've made the decision to just let conditional candidate generators work first, if they want to